gnome-extra/gnome-commander/gnome-commander-1.14.0.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+GNOME2_LA_PUNT="yes"
+
+inherit gnome2 optfeature
+
+DESCRIPTION="A graphical, full featured, twin-panel file manager"
+HOMEPAGE="https://gcmd.github.io/"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="chm exif gsf pdf samba taglib test +unique"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	app-text/yelp-tools
+	dev-libs/atk
+	dev-libs/glib:2
+	gnome-base/gconf:2
+	media-libs/fontconfig
```

Comment:
I see no evidence this is required.

Reply:
X86_64;/usr/lib64/gnome-commander/plugins/libfileroller.so;libfileroller.so;;libgtk-x11-2.0.so.0,libgdk-x11-2.0.so.0,libpangocairo-1.0.so.0,libatk-1.0.so.0,libcairo.so.2,libgdk_pixbuf-2.0.so.0,libgio-2.0.so.0,libpangoft2-1.0.so.0,libpango-1.0.so.0,libgobject-2.0.so.0,libglib-2.0.so.0,libharfbuzz.so.0,**libfontconfig.so.1**,libfreetype.so.6,libstdc++.so.6,libc.so.6,libgcc_s.so.1;x86_64

 TYPE   NEEDED FILE
ET_DYN libgtk-x11-2.0.so.0,libgdk-x11-2.0.so.0,libpangocairo-1.0.so.0,libatk-1.0.so.0,libcairo.so.2,libgdk_pixbuf-2.0.so.0,libgio-2.0.so.0,libpangoft2-1.0.so.0,libpango-1.0.so.0,libgobject-2.0.so.0,libglib-2.0.so.0,libharfbuzz.so.0,**libfontconfig.so.1**,libfreetype.so.6,libstdc++.so.6,libc.so.6,libgcc_s.so.1 /usr/lib64/gnome-commander/plugins/libfileroller.so